src/openforms/formio/datastructures.py

```diff
@@ -121,3 +133,12 @@ def __contains__(self, key: Hashable) -> bool:
         except PathAccessError:
             return False
         return True
+
+    def __iter__(self) -> Iterator[str]:
+        return iter(self._unnest(self.data))
```

Comment:
I don't think this is a good idea because it doesn't take editgrid into account at all.

I never had iterating over `FormioData` in mind as an intended usage, it was always intended to be used with the assumption that you know which key you need (so you know the key before-hand). I'm afraid this is blowing up the scope of this data structure way too much and will lead to other problems in the future.

Reply:
Hum makes sense, I added my reasoning in the meta issue, but I'll update it with this in mind